server/lib/currency.ts

```diff
@@ -269,3 +270,32 @@ export function reduceArrayToCurrency(array: AmountWithCurrencyAndDate[], curren
     },
   );
 }
+
+/**
+ * Gets exchange rate between given crypto currency and fiat.
+ */
+export async function getExchangeRate(fromCurrency: string, toCurrency: string): Promise<string> {
+  try {
+    if (SUPPORTED_CRYPTO_CURRENCIES.includes(fromCurrency) && !SUPPORTED_CRYPTO_CURRENCIES.includes(toCurrency)) {
+      const { rate: cryptoToUSDRate } = await getCryptoToUSDRate(fromCurrency);
```

Comment:
`getCryptoToUSDRate` should just returns the rate, instead of having to do `{ rate: cryptoToUSDRate }`.

Reply:
Done. 👍🏽